backend/Makefile

```
@@ -0,0 +1,8 @@
+run:
+	go run go/cmd/server/main.go
+
+test:
+	go test ./...
+
+lint:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)